src/modules/keyboardmanager/ui/EditShortcutsWindow.cpp

```diff
@@ -77,9 +78,20 @@ void createEditShortcutsWindow(HINSTANCE hInst, KeyboardManagerState& keyboardMa
         isEditShortcutsWindowRegistrationCompleted = true;
     }
 
-    // Find center screen coordinates
-    RECT desktopRect;
-    GetClientRect(GetDesktopWindow(), &desktopRect);
+    // Find coordinates of the screen where the settings window is placed.
+    HWND settingsWindow = GetForegroundWindow();
+    HMONITOR settingsMonitor = MonitorFromWindow(settingsWindow, MONITOR_DEFAULTTONULL);
+    RECT desktopRect{};
+    auto monitors = GetAllMonitorRects<&MONITORINFOEX::rcWork>();
+    for (const auto& monitor : monitors)
+    {
+        if (settingsMonitor == monitor.first)
+        {
+            desktopRect = monitor.second;
+            break;
+        }
+    }
```

Comment:
The above code duplicates code from EditKeyboardWindow so maybe it is a good idea to define a function `GetCurrentWindowMonitor` or something like that.